conf/messages.cy

```diff
@@ -122,33 +124,34 @@ destinationOffice.checkYourAnswersLabel = Were the goods received in Great Brita
 destinationOffice.error.required = Select if the goods were received in Great Britain or Northern Ireland
 destinationOffice.change.hidden = DestinationOffice
 
-selectReason.title.0 = Why are you alerting the consignor to an issue?
-selectReason.title.1 = Why are you rejecting the movement?
-selectReason.heading.0 = Why are you alerting the consignor to an issue?
-selectReason.heading.1 = Why are you rejecting the movement?
+selectReason.title.alert = Why are you alerting the consignor to an issue?
```

Comment:
Just searching the project for some of these messages, and can't find a few of them. Have I missed something, or should they be removed?

Reply:
@machinesleet Pushed a commit to fix that, that one should be used but was an incorrect on the view https://github.com/hmrc/emcs-tfe-alert-rejection-frontend/pull/22/commits/6aa50a2d858ef3bf2ab5bb9674e6a7fcb92e6552